docs/application/tizen-studio/release-notes/4-6-release-notes.md

```diff
@@ -0,0 +1,125 @@
+
+### Deprecated
+  - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.
+
+
+### Known Issues
+
+- Installer
+  - In macOS, Tizen Studio related file are marked as Malware, hence it is recommended to select **Anywhere** under allow apps as described in https://www.imore.com/how-open-apps-anywhere-macos-catalina-and-mojave
+- Tizen Studio
+  - Valgrind profiling is not supported in Tizen 6.5 platform
+  - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - In macOS from catalina and above versions, Native Templates-5.5 will not build with CLI when compiler is set to **gcc**.
+- Common
+  - If you install Tizen Studio in a directory that requires administrator privileges for access. For example, for **C:\ProgramFiles** location administrator privilege is needed to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you if you try to install it into such a directory.
+- Web and Native IDE
+  - You can create unit tests for Tizen 2.3.2 and higher version projects only. Unit testing for older versions is not supported.
+- Web IDE
+  - The preview tab in the Web Page Editor sometimes does not appear properly. Use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl + 4** of the Web SDK HTML Editor.
+  - In Rapid Development Support (RDS) mode, the web unit test result is not updated.
+  - Remote Inspector does not allow audit in the updated version of Chrome.
+- Certificate Manager
+  - Overwriting a duplicate certificate profile in the migration wizard works incorrectly on macOS.
+- Emulator
+  - In the Tizen 5.5 version, the emulator images app un-installation might take some time to complete if the app name contains more than 14 characters. A patch for the same is expected to be released in the next update.
+  - To use Tizen Emulator, use Intel VTx supported CPU and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for Tizen Emulator, see [Prerequisites for Tizen Studio](../setup/prerequisites.md).
+    - If the host machine is using NVIDIA&reg; Optimus&reg; technology on either Ubuntu or Windows, you must set the Tizen Emulator to run with your NVIDIA&reg; graphics card. For Ubuntu, verify the [bumblebee project](https://wiki.ubuntu.com/Bumblebee). For Windows, select **High Speed NVIDIA&reg; Processor** as **Preferred Graphics Processor** in the NVIDIA&reg; control panel.
```

Comment:
“The” has been removed before “Tizen Emulator” since it is recommended to not use “the” before nouns such as tool names. Also, "NVIDIA Control Panel" is a proper noun, so it should be capitalized. 
```suggestion
    - If the host machine is using NVIDIA&reg; Optimus&reg; technology on either Ubuntu or Windows, you must set Tizen Emulator to run with your NVIDIA&reg; graphics card. For Ubuntu, verify the [bumblebee project](https://wiki.ubuntu.com/Bumblebee). For Windows, select **High Speed NVIDIA&reg; Processor** as **Preferred Graphics Processor** in the NVIDIA&reg; Control Panel.
```